yarn-project/protocol-contracts/src/auth-registry/index.ts

```diff
@@ -7,7 +7,7 @@ import type { ProtocolContract } from '../protocol_contract.js';
 
 let protocolContract: ProtocolContract;
 
-export const AuthRegistryArtifact = loadContractArtifact(AuthRegistryJson as NoirCompiledContract);
+export const AuthRegistryArtifact: ContractArtifact = loadContractArtifact(AuthRegistryJson as NoirCompiledContract);
```

Comment:
Had to do this change to make it all compile.

I have no clue why it got triggered here but it looks harmless. Thought it's some issue with master but master builds so IDK.

<img width="1059" alt="image" src="https://github.com/user-attachments/assets/954f25f5-f561-4ec8-8043-6312da42c58c" />

Reply:
Wthat was the error?

Reply:
What's weird is that IDE derives the type when I hover over the constant.

![image](https://github.com/user-attachments/assets/93a14d58-ec6f-4d51-84d6-3a6d496fdc38)

Reply:
Yeah unsure, did some looking into this but couldn't find the root cause. Perhaps caused by a dependency cycle we have in the stdlib package. At any rate not a blocker.